### PR TITLE
chmod: match GNU error

### PR DESCRIPTION
**Pull request by deantvv**
Related to #2260

### CODE REVIEW

File: src/uu/chmod/src/chmod.rs [hunk 1]
```diff
@@ -262,8 +262,10 @@ impl Chmoder {
                         );
                     }
                     return Ok(());
+                } else if err.kind() == std::io::ErrorKind::PermissionDenied {
+                    show_error!("'{}': Permission denied", file.display());
                 } else {
-                    show_error!("{}: '{}'", err, file.display());
+                    show_error!("'{}': {}", err, file.display());
```
Comment (tertsdiepraam) [inline src/uu/chmod/src/chmod.rs:268]: Doesn't the `file.display()` need to be the second argument here?

**Review by tertsdiepraam — COMMENTED** [review]
Nice! Just one question about the second `show_error` call. Test looks good, too!

File: src/uu/chmod/src/chmod.rs (hunk 1 above)
Comment (deantvv) [reply src/uu/chmod/src/chmod.rs]: @tertsdiepraam Thanks for the catch. Fixed in new commit.

**Comment by sylvestre** [comment]
Improved the results:
https://github.com/uutils/coreutils-tracking

**Pull request merged**